packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -133,7 +134,16 @@ export const SelectItem = styled( Ariakit.SelectItem )`
 	justify-content: space-between;
 	padding: ${ ITEM_PADDING };
 	font-size: ${ CONFIG.fontSize };
-	line-height: 2.15rem; // TODO: Remove this in default but keep for back-compat in legacy
+	// Legacy line height is 28px.
+	// TODO: reassess for non-legacy v2
+	line-height: calc( 28px / ${ CONFIG.fontSize } );
```

Comment:
Apparently this `calc` [doesn't work](https://developer.mozilla.org/en-US/docs/Web/CSS/calc#rules_and_best_practices_while_using_calc) 😕 ("For `/`, the right operand must be unitless".)

Reply:
Maybe if we wrap `CONFIG.fontSize()` in a `stringToNumber()`? I appreciate the clarity that the explicit calc adds.

Reply:
To adhere more closely to legacy behavior, It's actually better to keep the hardcoded `28px` as V1 does, exactly for the reason you state. For example, in the default Storybook example of `CustomSelectControl` each select item has a custom `font-size`, but the line-height is supposed to stay the same.

Reply:
> the line-height is supposed to stay the same instead of scaling up/down.

Interesting - is this always the case? What if I have font-size 56px, won't it be ugly with line-height of 28px?

Reply:
I guess we can review that when assessing what `line-height` to adopt for the V2 — keeping it as hardcoded `28px` is probably the safest choice while swapping the V1 implementation